wolfi-baselayout.yaml

```diff
@@ -43,25 +43,17 @@ pipeline:
 
   - name: Install
     runs: |
-      for i in dev etc etc/profile.d etc/secfixes.d home proc root var/log usr/bin usr/local/lib sys tmp var/spool/cron opt run usr/lib; do
+      for i in dev etc etc/profile.d etc/secfixes.d home proc root var/log usr/bin sys tmp var/spool/cron opt run usr/lib; do
```

Comment:
dropping /usr/local/lib is not necessarily related here, is it?

Reply:
While true it isn't directly related, wolfi-baselayout isn't making usr/local directories for anything else and shouldn't need to. Some packages do create these directories for their own purposes. This felt like a useful cleanup to make. We could however leave it to a future cleanup.

Reply:
Essentially, people who use read-only mounts and read-only containers cannot bind mount files to that destination unless it exists.

Also i  am surprised we are getting away without usr/local/bin

Reply:
gotcha .... I remember seeing usr/local/bin somewhere else (not in wolfi-baselayout) and I don't remember where now